### ISSUE TITLE
label of input fields aren't clickable

### ISSUE THREAD

**Issue by pow-nico** [espocrm/espocrm#2739]
**Describe the bug**
Because the input fields aren't properly linked to their respective labels it is not possible to click at the label to set the keyboard focus into the input field

**To Reproduce**
1. open any site with an input field (e.g. login page)
2. click on a input field label

**Expected behavior**
After clicking on a label of an input field the keyboard focus should be set into that corresponding input field

**Screenshots**
![screen](https://github.com/espocrm/espocrm/assets/99261218/9f20bb64-188c-47a5-95ac-f715ba6e9a7b)

**EspoCRM version**
7.4.3

**Additional context**
This is an accessibility issue for blind and motorically impaired users (see WCAG) and an usability issue for power users

**Comment by pow-nico** [comment]
I hereby agree to the terms of the EspoCRM Contributor License Agreement v1.1.

2023-05-12

Nico Heinrich

**Comment by yurikuzn** [comment]
It's not that easy to do as we don't rely on IDs. There can be two forms on the page with same fields that will cause a conflict of IDs. I don't consider it to be a bug. Currently we feature frozen for a few months. Can't take a look into it.

**Comment by yurikuzn** [comment]
I can a TON of items in backlog. If this ever will be addressed, it can take a lot of time. No need to re-open as it's not easy to implement so that it to be applied for ALL forms. But I wouldn't like it to be inconsistent. I checked some systems and there was no this functionality too. I'm very busy these weeks, can't look into any more

**Comment by yurikuzn** [comment]
>  that's no problem as you also can nest any input field into a label element like this

It's even more problem to change the layout of the system that is used by thousands organizations.
